azure-user-guide/container-deployments/README.md

```diff
@@ -13,8 +13,6 @@ In addition, the DuploCloud platform has a built-in container management platfor
 DuploCloud supports many types of applications in Azure, including but not limited to:
 
 * Dockerized apps constitute about 90% of our user workloads. The platform orchestrates containerized application deployments using AKS or built-in container orchestrations as defined in the [Container orchestration features](../../gcp-user-guide/container-deployments/container-orchestrators.md) section.
-* Sagemaker
-* Glue&#x20;
 
```

Comment:
- Container Apps
- App Services
- VMs
- Blob Storage Accounts
- AGWs

This seems to be duplicative of https://docs.duplocloud.com/docs/azure-user-guide/azure-services.  Revisit flow of getting started vs conceptual overview